src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1829,8 +1917,12 @@ class NodeUtils(object):
         pbs.logmsg(pbs.EVENT_DEBUG4, '%s: numa nodes: %s' %
                    (caller_name(), self.numa_nodes))
         vnode_name = self.hostname
+
```

Comment:
I have avoided adding empty lines within methods up to this point. PEP8 allows it, but empty lines tend to become inconsistent over time.

Reply:
I have removed a few of these that are indeed unnecessary because the comment that introduces a new section of code.